buildscripts/hpat-conda-recipe/build.sh

```diff
@@ -3,3 +3,5 @@
 # OPENCV_DIR="${PREFIX}" DAALROOT="${PREFIX}"
 HDF5_DIR="${PREFIX}" MACOSX_DEPLOYMENT_TARGET=10.9 \
 $PYTHON setup.py build install --single-version-externally-managed --record=record.txt
+$PYTHON setup.py build_doc
+$PYTHON setup.py build_devdoc
```

Comment:
Could you please comment these lines out? I mean, do not delete them but comment out for a moment. We will need them later.